tests/local/conftest.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+import xarray as xr
+import numpy as np
+import pandas as pd
+
+@pytest.fixture()
+def create_local_netcdf(tmp_path_factory):
```

Comment:
I don't think this has to be a fixture. I guess for tests you want to produce several kinds of netcdf/geotiff files with varying dimensions, label ranges, etc
so you just need a helper function

Reply:
Done, fixture removed and created a helper function

Reply:
I would need `rioxarray` if I want to test also geoTIFFs. Does it make sense to include it in `tests_require` or not?  I wonder if these tests should run only if installing the `localprocessing` part?

Reply:
In principle, in CI we want to run all tests, but with localprocessing being quite heavy in terms of dependencies (all have to be installed with each run), this will make our CI a lot slower, so I'm not sure we should do that.
As long as it is an experimental feature we should skip the localprocessing test during the standard CI run and only run them manually as health check while development.

So in short: for now: keep all localprocessing related dependencies (also the ones just for testing) out of the standard requirements, and just put them in the localprocessing extra